service/api/src/main/java/org/eclipse/kapua/KapuaEntityNotFoundException.java

```diff
@@ -52,6 +52,12 @@ public KapuaEntityNotFoundException(String entityType, KapuaId entityId) {
         this.entityId = entityId;
     }
 
+    public KapuaEntityNotFoundException(String entityType) {
```

Comment:
Please remove this and check comment on [GwtJobTargetServiceImpl.java](https://github.com/eclipse/kapua/pull/2270/files#diff-bf02bba069510b97ad33fa57f2702882)